.travis.yml

```diff
@@ -23,18 +23,20 @@ install:
   - docker-compose exec web bash -c "rake db:setup"
   - docker-compose exec web bash -c "rake db:migrate"
   - docker-compose exec web bash -c "rake assets:precompile"
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then gem install danger danger-junit; fi'
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then bash -c "docker-compose exec web gem install danger danger-junit google-cloud-storage"; fi'
 
 script:
   - docker-compose exec web bash -c "wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb"
   - docker-compose exec web bash -c "dpkg -i google-chrome-stable_current_amd64.deb; apt-get -fy install"
-  - docker-compose exec web bash -c "wget https://chromedriver.storage.googleapis.com/73.0.3683.68/chromedriver_linux64.zip" # https://sites.google.com/a/chromium.org/chromedriver/
+  - docker-compose exec web bash -c "wget https://chromedriver.storage.googleapis.com/74.0.3729.6/chromedriver_linux64.zip" # https://sites.google.com/a/chromium.org/chromedriver/
   - docker-compose exec web bash -c "unzip chromedriver_linux64.zip"
   - docker-compose exec web bash -c "mv chromedriver /usr/local/bin/chromedriver"
   - docker-compose exec web bash -c "chmod +x /usr/local/bin/chromedriver"
+  - docker-compose exec web bash -c 'echo "$GOOGLE_APPLICATION_CREDENTIALS" > /tmp/credentials.json' # needs to be a path to a json file, so we shuffle...
+  - docker-compose exec web bash -c 'export GOOGLE_APPLICATION_CREDENTIALS="/tmp/credentials.json"'
```

Comment:
OK, these lines... they're not overwriting the env variable with the path as they should be...

```
[!] Invalid Dangerfile file: Unable to read the credential file specified by GOOGLE_APPLICATION_CREDENTIALS: file { does not exist
```